conan/internal/model/conf.py

```diff
@@ -129,6 +129,7 @@
     "tools.system.package_manager:mode": "Mode for package_manager tools: 'check', 'report', 'report-installed' or 'install'",
     "tools.system.package_manager:sudo": "Use 'sudo' when invoking the package manager tools in Linux (False by default)",
     "tools.system.package_manager:sudo_askpass": "Use the '-A' argument if using sudo in Linux to invoke the system package manager (False by default)",
+    "tools.system.pip_manager:python_interpreter": "Path to the Python interpreter to be used to create the virtualenv",
```

Comment:
```suggestion
    "tools.system.pipenv:python_interpreter": "Path to the Python interpreter to be used to create the virtualenv",
```
what about `tools.system.pipenv:python_interpreter` ?

Reply:
I used the same name as the python module, thinking it was our "naming standard". If we don't follow that pattern, I'll change it.

Reply:
I would say we don't have have a naming standard, In my opinion the most important rule is that is easy to find for users and the name is a bit self-explanatory to the extent possible? Check other confs to see if you think it could work